script/update-test-harness.js

```diff
@@ -42,8 +42,11 @@ else
   LZMA_CHECKSUM=$(shasum -a 256 $LZMA_FILE | awk '{print $1;}')
 fi
 
-GZIP_SIZE=$(du -h $GZIP_FILE | cut -f1)
-LZMA_SIZE=$(du -h $LZMA_FILE | cut -f1)
+GZIP_SIZE=\$(du -h $GZIP_FILE | cut -f1)
```

Comment:
This was a fix for an encoding issue (passing `$` through directly meant it was going to do some weird string interpolation because this is a JS script).